mg/builtins.go

```diff
@@ -74,12 +83,14 @@ func (bc BuiltinCmds) TypeCmd(bx *BultinCmdCtx) *State {
 	return bx.State
 }
 
-func (bc BuiltinCmds) EnvCmd(bx *BultinCmdCtx) *State {
+// EnvCmd finds all environment variables corresponding to bx.Args into the
+// bx.Output buffer.
+func EnvCmd(bx *BultinCmdCtx) *State {
 	buf := &bytes.Buffer{}
 	names := bx.Args
 	if len(names) == 0 {
 		names = make([]string, 0, len(bx.Env))
-		for k, _ := range bx.Env {
```

Comment:
Omitting `_` in func args and receivers is fine (I prefer the consistency, but have no strong opionion on it), but please keep them for ranges. It removes any confusion about what the variable holds and avoids bugs esp. considering channels returns the value by default.